ecs-templates.tf

```diff
@@ -6,7 +6,7 @@
 
 data "template_file" "beekeeper_path_scheduler_container_definition" {
   count    = var.instance_type == "ecs" ? 1 : 0
-  template = file("${path.module}/files/container-definition.json")
+  template = file("${path.module}/files/ecs-container-definition.json")
```

Comment:
This is more of a question on the general name changes here - are we only ever going to create `sns` resources for ecs deployments? In those cases what would be the drop in on kubernetes for those?

Reply:
I changed the name of "container-definition" to make it distinct from any k8s configuration.

With regards to SNS resources, do you mean the `operational-events` SNS topic? This is only needed for ECS as for ECS we set up some Cloudwatch dashboards which monitor the resources, and need a topic to send events to.